### PR TITLE
LPS-149121

### PR DESCRIPTION
**Pull request by p-albuquerque**
Related Issue: https://issues.liferay.com/browse/LPS-149121

### CODE REVIEW

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @p-albuquerque, can you please check if these failures are related to your changes? Thanks

**Comment by mateussgn** [comment]
ci:reevaluate:1347401_1489

**Comment by liferay-forms** [comment]
Hey @p-albuquerque, can you please check if these failures are related to your changes? Thanks

**Comment by mateussgn** [comment]
ci:test:relevant

**Comment by mateussgn** [comment]
LGTM! 🎉

**Comment by marcelabc** [comment]
Just started reviewing :)

**Review by marcelabc — CHANGES_REQUESTED** [review]
Hey, how are you doing?
I finished your review! Let me know if there is any question.

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectViewLocalServiceImpl.java [hunk 1]
```diff
@@ -337,9 +347,53 @@ private void _validate(long objectViewId, long objectDefinitionId)
 		}
 	}
 
+	private void _validate(
+			ObjectView objectView, List<ObjectViewColumn> objectViewColumns)
+		throws ObjectViewColumnFieldNameException {
+
+		Set<String> objectFieldNames = SetUtil.fromArray(new String[0]);
```
[the remark this answers is not in the document]
Comment (p-albuquerque) [reply modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectViewLocalServiceImpl.java:354]: okay, I'm changed for `Set<String> objectFieldNames = SetUtil.fromArray(new String[0]);` to `Set<String> objectFieldNames = new LinkedHashSet<>();` as I found in some situations searching

File: modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectViewLocalServiceTest.java [hunk 2]
```diff
@@ -264,14 +356,53 @@ private ObjectViewColumn _createObjectViewColumn(
...
 	}
 
+	private List<ObjectViewColumn>
+			_createObjectViewColumnsWithDuplicateObjectFieldName()
+		throws Exception {
+
+		ObjectViewColumn objectViewColumn1 = _createObjectViewColumn(
+			"Roger", "roger");
+
+		ObjectViewColumn objectViewColumn2 = _createObjectViewColumn(
```
[the remark this answers is not in the document]
Comment (p-albuquerque) [reply modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectViewLocalServiceTest.java:375]: Okay, got it

**Comment by p-albuquerque** [comment]
Resend in: #562

**Pull request closed without merging**